src/local/butler/reproduce.py

```diff
@@ -39,8 +38,8 @@
     urllib.urlencode({
         'scope':
             'email profile',
-        'client_id': ('981641712411-sj50drhontt4m3gjc3hordjmp'
-                      'c7bn50f.apps.googleusercontent.com'),
+        'client_id': ('602540103821-lccrsee9e5hbe3lpdsghin0'
```

Comment:
We need to remove the hardcode, can you add a TODO here. this should come from config or somewhere. can remove "# TODO(mbarbella): Don't use the old clusterfuzz-tools client id."

Reply:
In the most recent commit it's been changed to "# TODO(mbarbella): Client ID and domain should be configurable."